### PR TITLE
add feature: buttons positions timeseries and animatedtimeseries

### PR DESCRIPTION
**Pull request by matallo**
`animatedChanged` is currently empty and we're not using any of those attributes to know later if the widget is an animated time series or not, we only have a `type: times-series` in both cases

timeseries
7a0969ec-f0af-49e7-875f-3f6612cf4643

animatedtimeseries
8e6948b5-c8a0-4e19-8565-fed694ed31a9

CR @nobuti  /cc @xavijam

### CODE REVIEW

File: app/assets/stylesheets/editor-3/context-switcher.css.scss [hunk 1]
```diff
@@ -18,6 +18,9 @@
   right: 392px;
 }
 .Editor-contextSwitcher.has-timeSeries {
+  bottom: 167px;
+}
+.Editor-contextSwitcher.has-animatedtimeSeries {
```
Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/context-switcher.css.scss]: has-animatedTimeSeries?

Comment (xavijam) [inline app/assets/stylesheets/editor-3/context-switcher.css.scss]: Yeah

File: lib/assets/javascripts/cartodb3/data/widget-definitions-collection.js [hunk 2]
```diff
@@ -70,6 +70,12 @@ module.exports = Backbone.Collection.extend({
     return !!this.findWhere({ type: 'time-series' });
   },
 
+  isThereAnimatedTimeSeries: function () {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/data/widget-definitions-collection.js]: Hey, that function already exists!

`isThereTimeSeries`

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/data/widget-definitions-collection.js]: Ahhh, and animated, ok!.
Could we make a change in the other function, adding if we need the animated one or not, what do you think both?

Comment (nobuti) [inline lib/assets/javascripts/cartodb3/data/widget-definitions-collection.js]: 👍

File: lib/assets/javascripts/cartodb3/data/widget-definitions-collection.js [hunk 3]
```diff
@@ -66,8 +66,11 @@ module.exports = Backbone.Collection.extend({
     return this._attrsForThisTypeMap[newType](m);
   },
 
-  isThereTimeSeries: function () {
-    return !!this.findWhere({ type: 'time-series' });
+  isThereTimeSeries: function (opts) {
+    opts = opts || {};
+
+    var timeSeries = this.findWhere({ type: 'time-series' });
+    return opts.animated ? !!timeSeries : timeSeries && timeSeries.get('animated');
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/data/widget-definitions-collection.js]: 👍 

I'd personally do something like this if possible, not sure if all timeseries has the animated property to `false`/`true` by default:
```
var timeSeries = this.findWhere({ type: 'time-series', animated: !!opts.animated });
return !!timeSeries;
```

**Comment by matallo** [comment]
🚀

**Pull request merged**